AEPEdgeConsent.podspec

```diff
@@ -15,8 +15,8 @@ Pod::Spec.new do |s|
   s.swift_version = '5.1'
 
   s.pod_target_xcconfig = { 'BUILD_LIBRARY_FOR_DISTRIBUTION' => 'YES' }
-  s.dependency 'AEPCore', '>= 3.1.0'
```

Comment:
I see the current latest version to be `3.4.2`. Is there a `3.5.0` in pipeline?

Reply:
Yes. It includes the added APIs in Date+Format which Consent uses.